src/reply/reply.stories.tsx

```diff
@@ -0,0 +1,49 @@
+import { h } from 'preact'
+import { Reply } from './reply'
+import type { Meta, Story } from '@storybook/preact'
+
+export default {
+	title: 'Components/Reply Item',
+	component: Reply,
+	argTypes: {
+		replyTitle: {
+			name: 'Reply Title',
+			type: { name: 'string', required: true },
+			defaultValue: 'Hi Sally, I can do it for you!',
+		},
+		replyDate: {
+			name: 'Reply Date',
+			type: { name: 'string', required: true },
+			defaultValue: 'Today, 15:01',
+		},
+		replierName: {
+			name: 'Replier Name',
+			type: { name: 'string', required: true },
+			defaultValue: 'Ellis',
+		},
+		replierRep: {
+			name: 'Replier Reputation',
+			type: { name: 'number', required: true },
+			defaultValue: 52,
+		},
+		replyAmt: {
+			name: 'Reply Amount',
+			type: { name: 'number', required: true },
+			defaultValue: 12,
+		},
```

Comment:
Lets keep it as `number` now but we probably need to change the type to `BigNumber` or `bigint` depending on the implementation we choose.

Reply:
I'd probably tend towards `bigint` for futureproofing, even though it likely makes the bundle a bit larger right now.